tests/NA12878-chr14-AKT1_1000genomes-dragen-3.7.6/output_certified/results/multiqc/multiqc_data/multiqc_data.json

```diff
@@ -2,16 +2,16 @@
     "report_data_sources": {
         "Picard": {
             "InsertSizeMetrics": {
-                "NA12878-chr14-AKT1": "/home/ubuntu/2023/NPM-sample-qc/tests/NA12878-chr14-AKT1_1000genomes-dragen-3.7.6/work/45/e1eabb33a2af7f99fc04aea2d09873/NA12878-chr14-AKT1.insert_size_metrics.txt"
+                "NA12878-chr14-AKT1": "/home/ubuntu/2023/reads_mapped/NPM-sample-qc/tests/NA12878-chr14-AKT1_1000genomes-dragen-3.7.6/work/e5/1c7a6f5b55f96b2a43d4272222053b/NA12878-chr14-AKT1.insert_size_metrics.txt"
```

Comment:
I noticed that the `output_certified` folder has mentioned the path of temporary files.
Do we care about displaying a "clean" path ?
current commit: `home/ubuntu/2023/reads_mapped/NPM-sample-qc/tests/`
instead of ~cleaner: `/home/ubuntu/NPM-sample-qc/tests/`

Reply:
yes it is not very pretty ... but let's not care about this